docs/cloud/satellites.md

```diff
@@ -26,6 +26,17 @@ When using Earthly Satellites, even though the build executes remotely, the foll
 
 ![Satellite workflow](./img/satellite-workflow.png)
 
+Earthly Satellite instances come with their own cache volume. This means that performing builds repeatedly on the same satellite will result in faster builds. This can be especially useful when using Satellites from a sandboxed CI environment, where cache from previous builds would not otherwise be available. Below is a comparison with Earthly's [Shared remote cache features](../guides/shared-cache.md).
+
+| Cache characteristic | Satellite | Shared Cache |
+| --- | --- | --- |
+| Storage location | Satellite | A container registry of your choice |
+| Proximity to compute | ✔️ Same machine | ❌ Performing upload/download is required |
```

Comment:
ultra nit: is an EBS volume considered the same machine? AWS makes it feel as if it is performance-wise.

Reply:
Oh yeah, true. I would keep it as "implementation detail" and not even mention it, to keep this representation simple.